### PR TITLE
Make 'notebook runtimes' page disk-aware

### PR DESCRIPTION
**Pull request by panentheos**

### CODE REVIEW

**Comment by panentheos** [comment]
Marking this reviewable. There's one TODO remaining that I think could use some group brainstorming, but it should be good otherwise.

File: src/pages/Clusters.js [hunk 1]
```diff
@@ -52,88 +129,171 @@ const Clusters = () => {
...
+            cellRenderer: ({ rowIndex }) => {
+              return formatUSD(persistentDiskCostMonthly(filteredDisks[rowIndex]))
+            }
+          },
+          {
+            size: { basis: 50, grow: 0 },
+            headerRenderer: () => null,
+            cellRenderer: ({ rowIndex }) => {
+              const { id, status } = filteredDisks[rowIndex]
+              // TODO PD: there should be some way of identifying which disk is connected to which runtime
```
Comment (breilly2) [inline src/pages/Clusters.js:282]: Ah, interesting. One quick thought that doesn't quite address this is to add some icons to at least indicate whether or not a runtime has a disk attached and whether or not a disk is attached to a runtime. Currently the only way to see that it either in the runtime delete modal or whether or not the disk delete button is disabled.

**Review by breilly2 — APPROVED** [review]
👍  I checked it out in browser with both an attached disk and a floating disk and all looks good. I didn't test exhaustively... and only with the ajax overrides, not against a real leonardo.

File: src/pages/Clusters.js (hunk 1 above)
Comment (panentheos) [reply src/pages/Clusters.js:282]: Yeah, one potential idea is to add icons, and let people click on them to highlight the linkages between rows. Not sure if that's a bit too weird though.

**Pull request merged**